daffodil-test/src/test/resources/edu/illinois/ncsa/daffodil/section06/namespaces/namespaces.tdml

```diff
@@ -1549,7 +1549,12 @@
       <tdml:error>MinInclusive(16) must be less than or equal to MaxInclusive(10)</tdml:error>
       <tdml:error>shishi</tdml:error>
       <tdml:error>Location line</tdml:error>
-      <tdml:error>section06/namespaces/shi</tdml:error>
+      <!-- 
```

Comment:
See comment in TestJavaAPI.java - this change is undesirable. If we can't avoid it we can live with it, but really we don't want to lose precision on the error locations just because we're resolving the properties differently now.